### PR TITLE
update spec status

### PR DESCRIPTION
**Pull request by mkhraisha**
update Readme to point to new Spec

### CODE REVIEW

**Review by msporny — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,11 +5,16 @@ high-performance mechanism for publishing the revocation status of Verifiable
...
 mailing list as well.
 
 ### Other useful links
-* [Public group email archive](https://lists.w3.org/Archives/Public/public-credentials/)
+
+- [Public group email archive](https://lists.w3.org/Archives/Public/public-credentials/)
+
+### Specification Status
+
+- This specification has been superceeded by [VC Status List 2021](https://w3c-ccg.github.io/vc-status-list-2021) and will no longer be maintained
```
Comment (msporny) [inline README.md]:
```suggestion
- This specification has been superseded by [VC Status List 2021](https://w3c-ccg.github.io/vc-status-list-2021) and will no longer be maintained
```

**Review by msporny — CHANGES_REQUESTED** [review]
Fixed typo, other than that, looks good... thank you for the PR @mkhraisha !

The spec itself should probably do a meta-refresh to redirect people to the new spec... I don't think we even want people reading the old spec.

```
    <meta http-equiv="refresh" content="0;URL=''https://w3c-ccg.github.io/vc-status-list-2021/" />    
```

If you made the change above in this PR, I'd pull it in.

File: README.md (hunk 1 above)
Comment (TallTed) [reply README.md]: Sentence should also get a concluding "."....

**Comment by mkhraisha** [comment]
Sorry for delay this got lost, updated in your change

**Pull request merged**